missilewars-plugin/src/main/java/de/butzlabben/missilewars/player/PlayerData.java

```diff
@@ -38,25 +39,39 @@
 @AllArgsConstructor
 public class PlayerData implements ConfigurationSerializable {
 
-    private final long time;
     private UUID uuid;
     private ItemStack[] contents;
-    private float exp;
+    private GameMode gameMode;
     private double health;
+    private float exp;
     private int expLevel, foodLevel;
-    private GameMode gameMode;
+    private final long time;
 
     public PlayerData(Player player) {
+        uuid = player.getUniqueId();
         contents = player.getInventory().getContents();
+        gameMode = player.getGameMode();
+        health = player.getHealth();
         exp = player.getExp();
         expLevel = player.getLevel();
         foodLevel = player.getFoodLevel();
-        health = player.getHealth();
-        gameMode = player.getGameMode();
-        uuid = player.getUniqueId();
         time = System.currentTimeMillis();
     }
 
+    /**
+     * This method is used to load the original player data from the temporary player-data file.
+     */
+    public PlayerData(Map<String, Object> elements) {
+        uuid = UUID.fromString(elements.get("uuid").toString());
+        contents = ((ArrayList<ItemStack>) elements.get("contents")).toArray(new ItemStack[]{});
```

Comment:
```suggestion
        contents = ((List<ItemStack>) elements.get("contents")).toArray(new ItemStack[] {});
```
Always cast against Interfaces, the library for example could change from a ArrayList to a LinkedList, also note the new space at the end (formatting)